.github/workflows/reusable-change-detection.yml

```
@@ -126,13 +126,17 @@ jobs:
           .github/workflows/reusable-docs.yml
         format: csv  # works for paths with spaces
     - name: Check for docs changes
+      # We only want to run this on PRs when related files are changed,
+      # or when user triggers manual workflow run.
       if: >-
-        github.event_name == 'pull_request'
-        && steps.changed-docs-files.outputs.added_modified_renamed != ''
+        (github.event_name == 'pull_request'
+          && steps.changed-docs-files.outputs.added_modified_renamed != ''
```

Arguably, cleaner formatting is
```suggestion
        (
          github.event_name == 'pull_request'
          && steps.changed-docs-files.outputs.added_modified_renamed != ''
```